tests/deploy/dgd_utils.py

```
@@ -1722,6 +1760,47 @@ def port_forward(
+            except (requests.ConnectionError, httpx.TransportError) as error:
+                if attempt == 1:
+                    raise
+
+                self._logger.warning(
+                    "Frontend request transport failed; rebuilding the pod "
+                    "port-forward and retrying once: %s",
+                    error,
+                )
+                try:
+                    active_port_forward.stop()
+                except Exception as stop_error:
```

Ruff (0.16.3)

[warning] Do not catch blind exception: `Exception`

(BLE001)